genome.spec

```diff
@@ -181,36 +195,65 @@ typedef structure {
 typedef int Bool;
 
 /*
-Genome object holds much of the data relevant for a genome in KBase
-    Genome publications should be papers about the genome
-Should the Genome object contain a list of contig_ids too?
-Source: allowed entries RefSeq, Ensembl, Phytozome, RAST, Prokka, User_upload
-    #allowed entries RefSeq, Ensembl, Phytozome, RAST, Prokka,
-User_upload controlled vocabulary managed by API
-
-Domain is a controlled vocabulary
-Warnings : mostly controlled vocab but also allow for unstructured
-Genome_tiers : controlled vocabulary (based on ap input and API checked)
-Allowed values: #Representative, Reference, ExternalDB, User
-
-Examples Tiers:
-All phytozome - Representative and ExternalDB
-Phytozome flagship genomes - Reference, Representative and ExternalDB
-Ensembl - Representative and ExternalDB
-RefSeq Reference - Reference, Representative and ExternalDB
-RefSeq Representative - Representative and ExternalDB
-RefSeq Latest or All Assemblies folder - ExternalDB
-User Data - User tagged
-
-Example Sources:
-RefSeq, Ensembl, Phytozome, Microcosm, User, RAST, Prokka, (other annotators)
-
+Field descriptions:
+    id: string - object id
```

Comment:
This isn't an object ID. Not sure what it is

Reply:
Will ask jason about this